Description:  Fix the FTBFS with clang. The non-void function should return a value in autolog.c
Author: Arthur Marble <arthur@info9.net>
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=740031
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/

--- a/autolog.c
+++ b/autolog.c
@@ -355,7 +355,7 @@
 
     if (!(f=fopen(datfname,"w+"))) {
 	bailout("Can't create data-file.", 6);
-	return;
+	return 0;
     }
 
     fprintf(f,"%d\n",userfill);
@@ -376,7 +376,7 @@
     FILE* f;
 
 /*.. if no table exists, no problem, just return. ...........................*/
-    if (!(f=fopen(datfname,"r"))) { return; }
+    if (!(f=fopen(datfname,"r"))) { return 0; }
 
 /*.. check, how many lines with data will follow. ...........................*/
     fscanf(f,"%d",&userfill);
@@ -1041,7 +1041,7 @@
 /*.. have ps tell us all current users, uids and pids. ......................*/
     if (!(ps = popen(ps_cmd, "r")) ){
 	bailout("Can't use ps program", 6);
-	return;
+	return 0;
     }
     fgets(iline, LINELEN, ps);		 /* get header-line */
 
@@ -1090,7 +1090,7 @@
         if(!fgets(iline, LINELEN, ps)) {
           /* end of file, exit*/
           fclose(ps);
-          return;
+          return 0;
         }
         i=strlen(iline);
     }
